segments/battery.py

```diff
@@ -0,0 +1,28 @@
+def add_battery_segment():
+    CAP_FILE = '/sys/class/power_supply/BAT0/capacity'
+    STATUS_FILE = '/sys/class/power_supply/BAT0/status'
+    LOW_BATTERY_THRESHOLD = 20
+    
+    f = open(CAP_FILE)
+    cap = f.read().strip()
+    f.close()
```

Comment:
Instead of opening/closing the file this way, the Pythonic way is:

```python
with open(CAP_FILE) as f:
    cap = f.read().strip()
```

Same for STATUS_FILE